sympy/series/gruntz.py

```diff
@@ -550,6 +550,11 @@ def mrv_leadterm(e, x):
     #
     w = Dummy("w", positive=True)
     f, logw = rewrite(exps, Omega, x, w)
+
+    # Ensure expression of the form exp(log(...)) din't get simplified automatically in the previous steps.
```

Comment:
Need to fix "din't" 
Maybe doesn't can be used ?
or something like 
Ensure expressions of the form exp(log(...)) don't get simplified automatically in the previous steps.

Reply:
Also not sure if the failing check is a blocker here. Maybe a squash and rebase would be better